backend/dal/requestDAL.go

```diff
@@ -44,7 +44,7 @@ func (dal *RequestDAL) GetLastRequests(before time.Time, size int) ([]models.Req
 }
 
 // InsertRequestInDB Receives a request as a parameter and inserts into the database
-func (dal *RequestDAL) InsertRequestInDB(request *models.Request) error {
+func (dal *RequestDAL) InsertRequestInDB(request *models.Request) (string, error) {
```

Comment:
Same thing as InsertOfferInDB function: Add comment explaining the return parameters

Reply:
Resolved in commit 222813e